pubsub/tests/unit/pubsub_v1/publisher/batch/test_thread.py

```diff
@@ -34,13 +34,15 @@ def create_client():
     return publisher.Client(credentials=creds)
 
 
-def create_batch(autocommit=False, **batch_settings):
+def create_batch(autocommit=False, topic="topic_name", **batch_settings):
     """Return a batch object suitable for testing.
 
     Args:
         autocommit (bool): Whether the batch should commit after
             ``max_latency`` seconds. By default, this is ``False``
             for unit testing.
+        autocommit (topic): The name of the topic the batch should publish
```

Comment:
This looks wrong. Should it not be `topic (str): ...`?

Reply:
Somebody throw a rubber chicken at me, that was indeed the original intention, yes.

Reply:
🐔